geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -6383,6 +6393,9 @@ protected void distributeDestroyRegion(RegionEventImpl event, boolean notifyOfRe
    */
   @Override
   public void postCreateRegion() {
+    // add region level tombstone sweeper
+    this.cache.getTombstoneService().addRegionTombstoneSweeper(this);
```

Comment:
Does this get created for every bucket region too, or just the PR?
Shouldn't there instead be a way to configure on a per region basis?